### DISCUSSION TITLE
Is it possible to change the default hostname "a0d7b954-ssh"?

### DISCUSSION THREAD

**Discussion by diegofrances** [hassio-addons/addon-ssh#706] [Q&A]
Hello,

I would like to know if possible to change the default plugin hostname "a0d7b954-ssh" to something else.

Thank you in advance.

**Comment by frenck** [comment]
Nope.

../Frenck

**Comment by diegofrances** [comment]
Have you considered to provide this feature in a future release or is the name code dependant?

Thank you in advance.

**Reply by frenck** [reply]
This is not managed or in control of an add-on.

**Reply by diegofrances** [reply]
Do you mean that this is part of the HAOS?

**Reply by frenck** [reply]
Yes, each add-on get assigned a hostname based on the repository hash (`a0d7b954` for all community add-ons) and the add-on slug (`ssh`).
